From 295ae5cb8198ea6dc45fc176ba9b2ee649997f2d Mon Sep 17 00:00:00 2001
From: Jan Zeleny <jzeleny@redhat.com>
Date: Wed, 30 Jul 2014 10:05:56 +0200
Subject: [PATCH] Changed script paths in __os_install_post

Required to keep up with rpm and the current system layout
---
 macros.scl | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/macros.scl b/macros.scl
index 56d85c9ed7af098e138704a5267beaf86d8f76ba..d888e01a22acc2b2cb8ea56ebeae6814f4ffa721 100644
--- a/macros.scl
+++ b/macros.scl
@@ -78,12 +78,12 @@ package or when debugging this package.
 %scl_debug
 %global __os_install_post %{expand:
     /usr/lib/rpm/brp-scl-compress %{_scl_root}
-    %{!?__debug_package:/usr/lib/rpm/redhat/brp-strip %{__strip}
-    /usr/lib/rpm/redhat/brp-strip-comment-note %{__strip} %{__objdump}
+    %{!?__debug_package:/usr/lib/rpm/brp-strip %{__strip}
+    /usr/lib/rpm/brp-strip-comment-note %{__strip} %{__objdump}
     }
-    /usr/lib/rpm/redhat/brp-strip-static-archive %{__strip}
+    /usr/lib/rpm/brp-strip-static-archive %{__strip}
     /usr/lib/rpm/brp-scl-python-bytecompile %{__python} %{?_python_bytecompile_errors_terminate_build} %{_scl_root}
-    /usr/lib/rpm/redhat/brp-python-hardlink
+    /usr/lib/rpm/brp-python-hardlink
     %{!?__jar_repack:/usr/lib/rpm/redhat/brp-java-repack-jars}
 %{nil}}
 BuildRequires: scl-utils-build
-- 
1.9.3

